development.md

```diff
@@ -36,23 +36,23 @@ Generally, internal correctness is verified via _unit tests_ and correctness wit
 
 Unit tests run against every version of Capybara _at most once_ and against both the minimum and maximum Ruby version _at least once_.
 
-Integration tests run against every version of Rails at least once and every version of Capybara at least once. 
+Integration tests run against every version of Rails at least once and every version of Capybara at least once.
 
 ## Dependencies
 
-AePageObjects is tested across various versions of Capybara, Ruby, and Rails. 
+AePageObjects is tested across various versions of Capybara, Ruby, and Rails.
```

Comment:
I think we only test with one version of Capybara?

Reply:
I think it has the potential to test against multiple, but we just coincidentally only test agains `~> 3`.